src/content/perf/impeller.md

```diff
@@ -99,16 +99,15 @@ app's `Info.plist` file.
 
 ### Android
 
-As of Flutter 3.16, Impeller is available behind
-a flag on Android devices that support Vulkan.
-It continues to be in preview as of the 3.19 release.
+Impeller is available behind a flag on Android
+devices that support Vulkan.
```

Comment:
**chore:** Provide the explanation of "preview" that the original issue requested.

```suggestion
On Android devices that support Vulkan,
you can test and experiment with Impeller.

Impeller might not be suitable for production
environments at this time. 
```